.github/workflows/du.yml

```
@@ -0,0 +1,29 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: DeterminateSystems/nix-installer-action@main
+      - name: Update go.sum
+        run: |
+          nix run .#goSumUpdate
+      - name: Commit flake.lock
+        run: |
+          nix flake update --commit-lock-file
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.4)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)